_posts/2022-04-27-newpipe-0.23.0.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "NewPipe 0.23.0 released: Behold the behemoth"
+short: "NewPipe 0.23.0 released"
+date: 2022-04-27 8:00:00 + 01:00
+categories: [pinned, release]
+author: <a href="https://github.com/opusforlife2">@Opusforlife</a> & <a href="https://github.com/poolitzer">@Poolitzer</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+This release got delayed by quite a bit, and so it just grew and grew and grew. You're getting a truly massive release to make up for the long wait. Be ready for a whopper of a blog post!
+
+<!-- more -->
+
+Wow. Where do we even start? Let's see... new stream notifications! Now your phone can ping you even more frequently than before! Now you get seamless transition to and from background! Now there is no pause in the audio when switching between apps! YouTube/YouTube Music Mixes, the app suggests you these in the related items so that you can binge curated content to your hearts'... content. And finally, the popup player is usable again on Android 12!
```

Comment:
I'd remove the "Now you get seamless transition to and from background!", since it is misleading, as people might think that *switching* players is instant, while it is not. "Now there is no pause in the audio when switching between apps!" is the correct thing to say. 

I would also remove the Mixes news, since YouTube has already broken them :-/

```suggestion
Wow. Where do we even start? Let's see... new stream notifications! Now your phone can ping you even more frequently than before! Now there is no pause in the audio when switching between apps! And finally, the popup player is usable again on Android 12!
```

Reply:
> I would also remove the Mixes news, since YouTube has already broken them :-/

@opusforlife2 said it works for them, so it should be only an issue in the EU.

Reply:
Oh, right, I will change my suggestions